app/views/apps/child_birth_app/picking_up_protocol/ask_widow.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_tag apps_child_birth_app_picking_up_protocol_path, method: :get, remote: true do %>
+      <%= hidden_field_tag :mother_civil_state, 'widow' %>
+      <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset" aria-describedby="example-hint">
+          <legend class="govuk-fieldset__legend">
+            Narodilo sa dieťa do 300 dní od úmrtia manžela?
```

Comment:
netreba tam dat `alebo manzelky` ?

Reply:
Je to cisto len pre vdovy (urcuje sa to podla stavu matky). Kadozpadne obsahovo toto este nie je hotove. V tomto PR chceme mat hlavne porieseny mechanizmus embed appiek.